x-pack/plugins/rollup/public/crud_app/app.js

```diff
@@ -12,7 +12,7 @@ import { JobList } from './sections';
 export const App = () => (
   <div>
     <Switch>
-      <Route path={CRUD_APP_BASE_PATH} component={JobList} />
+      <Route exact path={CRUD_APP_BASE_PATH} component={JobList} />
```

Comment:
not familiar with this attribute, can you give me some insight on what it does and why we need it?

Reply:
It will become more important once we add the wizard. With `exact` the router will only match against `/rollup_jobs`. If we left it out then the router would also match paths beginning with this string, e.g. `rollup_jobs/create` ([docs[(https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Route.md#exact-bool)).